### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by lazyCode-404**
### Things implemented in this milestone:
- Set up a new project with webpack 
- Create a To Do List

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+- html
+- css
+- javascript
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+git clone  
+Main
+cd list-structure
```
Comment (Mre55) [inline README.md]: Kindly edit your `Getting Started` section to give brief steps on how to clone and run the project properly. It's should be something like this:

```
git clone git@github.com:BwalyaNonde/to-do-list.git
cd to-do-list
npm install
npm run build
npm start
```
Then check `http://localhost:8080/`
This is the proper way of cloning the project and running it.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @BwalyaNonde,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 

- [ ] Please add `.gitignore` file to your project and add `node_modules` to it in order to ignore it.

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (lazyCode-404) [reply README.md]: @Mre55 Thank you so much for your review and making me a better developer! 👍 

Happy Coding!!

**Comment by lazyCode-404** [comment]
@chaw-bot Thank you for approving my project 👍

**Pull request merged**